wasmcloud-test-util/Cargo.toml

```diff
@@ -10,10 +10,10 @@ repository = "https://github.com/wasmcloud/wasmcloud-test"
 readme = "README.md"
 
 [dependencies]
-smithy-bindgen = "0.1.0"
 serde_bytes = "0.11"
 regex = "1"
 wasmbus-rpc = { version = "0.14", features = ["otel"]}
+wasmcloud-interface-testing = "0.8.0"
```

Comment:
This uses wasmbus-rpc 0.13, doesn't it? I'm surprised there wasn't a conflict here, and are we sure it doesn't create one downstream?

- [ ] double check consumers of this crate can rely on wasmbus-rpc 0.14 and this branch of wasmcloud-test-util

Reply:
Looks like there are two public consumers of `wasmcloud-interface-testing`, `wash-cli` and `wasmcloud-provider-httpserver` -- right now `wasmcloud-provider-httpserver` also uses `wasmbus-rpc` version `0.13` -- you're right that there is at least one conflict though, but it has to do with `async-nats`:

```
error[E0599]: no method named `with_client_auth_cert` found for struct `ConfigBuilder` in the current scope
   --> ~/.cargo/registry/src/index.crates.io-6f17d22bba15001f/async-nats-0.30.0/src/tls.rs:117:29
    |
117 |                     builder.with_client_auth_cert(cert, key).map_err(|_| {
    |                             ^^^^^^^^^^^^^^^^^^^^^ help: there is a method with a similar name: `with_no_client_auth`

For more information about this error, try `rustc --explain E0599`.
error: could not compile `async-nats` (lib) due to previous error
warning: build failed, waiting for other jobs to finish...
```

Also, turns out [`wasmcloud-test-util` is the only thing that depends on `wasmbus` `0.14` right now](https://crates.io/crates/wasmbus-rpc/reverse_dependencies).